devel/gnu-extension-headers/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gnu-extension-headers
+version             1.0
+
+categories          lang devel
+platforms           darwin
+license             GPL-3+
+
+maintainers         nomaintainer
+
+description         Provides versions of various GNU extension headers
+long_description    ${description}
+
+livecheck.type      none
+
+fetch.type          none
+use_configure       no
```

Comment:
Usually when this is set to no, changes need to be made to support the universal variant. Or maybe just disable it.

Reply:
Or maybe you just need to add:

`supported_arches noarch
`

Reply:
Yeah, i think noarch is the way here...

Reply:
done